src/platform/pdf/test/templates/medical_records/medical_records.unit.spec.js

```diff
@@ -100,26 +100,6 @@ describe('Medical records PDF template', () => {
         }
       }
 
-      expect(artifactCount).to.eq(10);
-    });
-
-    it('Horizontal rules below result sections may be suppressed', async () => {
```

Comment:
I'd like to not remove the functional tests here if possible?

Reply:
ok, I looked a little closer and realized that this PR actually removes two artifact tags - I don't believe that removing the artifact tags altogether is the correct solution here.

@SarahKay8 can you clarify? My understanding is that decorative elements that should not be announced to the user should have an Artifact tag. Alternatively, should they be read out as separator elements? (since they have semantic meaning in terms of reinforcing the distinction between multiple items)

Reply:
To clarify, I'm thinking the issue is with the multiple levels of Artifact tags.

Reply:
@acrollet
	
If the decorative element has a meaningful purpose, such as separating content or providing visual structure, we can use a separator element with appropriate semantic information.


Example- 
<hr role="separator">

the <hr> (horizontal rule) element is given a role attribute to explicitly indicate its purpose as a separator.

@acrollet

Reply:
if the decorative element serves a purely visual purpose and its announcement could potentially be distracting or confusing, use an artifact tag

Reply:
Thank you @SarahKay8 ! Which pdf tag and/or descriptive text would be most appropriate to convey this?

@BobbyBaileyRB do you agree that these separators have semantic meaning in this context?

Reply:
##For visual separators

we can use the` <HR> (horizontal rule)` tag with an `aria-label` 
Example:

`<HR role="separator" aria-label="Section Divider">
`
	the aria-label attribute provides descriptive text for screen readers.

Or we can use CSS
 (Paragraph) with Borders:

	•	You can use a paragraph` (<P>) `element with CSS styling for borders to create a visual separator.


`<P style="border-top: 1px solid #000;">Section Separator</P>
`


##Artifacts for Decorative Elements:
	
If the visual structure is purely decorative and doesn’t convey specific information, you can use an `<Artifact>` tag.

Something like - 

`<Artifact>This line is decorative and does not convey additional information.</Artifact>
`